### PR TITLE
Feedback

### PR DESCRIPTION
**Pull request by faloi**
:wave:! GitHub Classroom created this pull request as a place for your teacher to leave feedback on your work. It will update automatically. **Don’t close or merge this pull request**, unless you’re instructed to do so by your teacher.

In this pull request, your teacher can leave comments and feedback on your code. Click the **Subscribe** button to be notified if that happens.

Click the **Files changed** or **Commits** tab to see all of the changes pushed to `master` since the assignment started. Your teacher can see this too.

<details>
<summary>
<strong>Notes for teachers</strong>
</summary>

Use this PR to leave feedback. Here are some tips:

- Click the **Files changed** tab to see all of the changes pushed to `master` since the assignment started. To leave comments on specific lines of code, put your cursor over a line of code and click the blue **+** (plus sign). To learn more about comments, read “[Commenting on a pull request](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request)”.
- Click the **Commits** tab to see the commits pushed to `master`. Click a commit to see specific changes.
- If you turned on autograding, then click the **Checks** tab to see the results.
- This page is an overview. It shows commits, line comments, and general comments. You can leave a general comment below.

For more information about this pull request, read “[Leaving assignment feedback in GitHub](https://classroom.github.com/help/leaving-feedback-in-github)”.
</details>

Subscribed: @rootm3

### CODE REVIEW

**Comment by rootm3** [comment]
@faloi hola profe, ahi termine de subirlo, si hay algo que corregir me avisa, Pablo

File: README.md [hunk 1]
```diff
@@ -23,3 +23,18 @@ Y de yapa, dos ejemplos de cómo incluir una imagen en Markdown (vean este archi
 ![Directamente desde un link](http://www.plantuml.com/plantuml/png/JSqn2W9138NXFgT8gLUmMb5hlS9Z9gu3cGmaMH74kxl25lR_zzuK2S-bAnL51DyKC_XDpEcmw2tbOdVnBXLrjp_mKUT17zhoqrBWA0y4ool5wl-2jQVe6TR63rxJnuQdDLmllm00)
 
 ![Desde un archivo local](./images/ejemplo-uml.png)
+
+
+**Subida de diagramas**
```
Comment (faloi) [inline README.md:28]: En el diagrama van solo las _firmas_ de los métodos: nombre, parámetros, tipo que devuelven. No ponemos detalles de implementación.

File: src/barrio.wlk [hunk 2]
```diff
@@ -1,21 +1,27 @@
...
+	//Lista:  Huerta, Mascota y Hogar
 	const property elementos = []
-
+	
+	//No devuelve nada solo agrega obj a la lista elementos (seria un void en java)
 	method agregarElemento(elemento) {
 		elementos.add(elemento)
 	}
 
+	//Lo mismo que arriba pero al revés
```
Comment (faloi) [inline src/barrio.wlk:11]: Ojo, solo pedimos poner qué devuelve (o nada, en este caso). La indicación "lo mismo que arriba" no está buena en ese sentido.

File: src/elementos.wlk [hunk 3]
```diff
@@ -1,41 +1,41 @@
...
-	var property capacidadProduccion
+	var property capacidadProduccion //es un int
 
-	method esBueno() {
+	method esBueno() { //retorna un boolean
 		return capacidadProduccion > INTA.produccionMinima()
 	}
 
-	method fueAtacado(plaga) {
+	method fueAtacado(plaga) { //no devuelve nada //idem arriba
```
Comment (faloi) [inline src/elementos.wlk:38]: Ya acá se te descontroló el "idem", no es claro a qué se refiere. A veces un poco de copy-paste no está mal y ayuda a entender... :smirk:

**Pull request closed without merging**